cypress/e2e/pages/workProgressSummary.cy.ts

```diff
@@ -1,20 +1,83 @@
+          cy.findByTestId('type').select('Work Type');
+          cy.get('[data-testid = valueSelect]')
+            .children('option')
+            .then(($options) => {
+              const optionValues = $options.toArray().map((elem) => elem.label);
+              expect(optionValues).to.deep.eq([
+                'Work Type 1',
+                'Work Type 2',
+                'Work Type 3',
+                'TEST_WT_1',
+                'Work Type 5'
+              ]);
+            });
+        });
+      });
+    });
   });
-  it('it should display Work Type, status, number of work requests and total number of blocks, slides and original samples required columns', () => {
-    cy.findByRole('table').get('th').should('have.length', 6);
-    cy.findByRole('table').get('th').eq(0).should('have.text', 'Work Type');
-    cy.findByRole('table').get('th').eq(1).should('have.text', 'Status');
-    cy.findByRole('table').get('th').eq(2).should('have.text', 'Number of Work Requests');
-    cy.findByRole('table').get('th').eq(3).should('have.text', 'Total Number of Blocks');
-    cy.findByRole('table').get('th').eq(4).should('have.text', 'Total Number of Slides');
-    cy.findByRole('table').get('th').eq(5).should('have.text', 'Total Number of Original Samples');
+
+  describe('Work progress summary table', () => {
+    before(() => {
+      cy.visit('./work_progress_summary');
+    });
+
+    it('should display summary table', () => {
+      cy.findByRole('table').should('be.visible');
+    });
+
+    it('it should display Work Type, status, number of work requests and total number of blocks, slides and original samples required columns', () => {
+      cy.findByRole('table').get('th').should('have.length', 6);
+      cy.findByRole('table').get('th').eq(0).should('have.text', 'Work Type');
+      cy.findByRole('table').get('th').eq(1).should('have.text', 'Status');
+      cy.findByRole('table').get('th').eq(2).should('have.text', 'Number of Work Requests');
+      cy.findByRole('table').get('th').eq(3).should('have.text', 'Total Number of Blocks');
+      cy.findByRole('table').get('th').eq(4).should('have.text', 'Total Number of Slides');
+      cy.findByRole('table').get('th').eq(5).should('have.text', 'Total Number of Original Samples');
+    });
+
+    context('Filtered table', () => {
+      it('correctly applies the filters on the table data', () => {
+        cy.visit('./work_progress_summary');
+        cy.findByTestId('type').select('Status');
+        cy.get('[data-testid = valueSelect]').select('active');
+        cy.findByRole('button', { name: /Search/i }).click();
+        cy.findByRole('table').contains('active');
+      });
+
+      it('correctly applies the filters on the table data when given through a url param', () => {
+        cy.visit('./work_progress_summary?searchType=Work%20Type&searchValues[]=Work%20Type%201');
+        cy.findByRole('table').contains('Work Type 1');
+      });
```

Comment:
Also, good to check it is only displaying 'Work Type 1'  and not displaying any other work types

Reply:
Yep will add check to make sure there aren't other work types